packages/shared/src/consts.ts

```diff
@@ -87,3 +87,5 @@ export const MIN_TICK = -887272;
 export const MAX_TICK = -MIN_TICK;
 
 export const FULL_TICK_RANGE = { start: MIN_TICK, end: MAX_TICK };
+
+export const RECOMMENDED_SLIPPAGE_TOLERANCE_PERCENTAGE = 2;
```

Comment:
maybe making this an ENV variable would be better

Reply:
we discussed it, don't think it brings much since we want to replace it quite soon with a dynamic value but @niklasnatter can weigh in

Reply:
Dynamic coming from quoter endpoint? (aka https://chainflip-swap.chainflip.io/v2/quote)

Reply:
fine to have a constant for now - we will replace it with something dynamic relatively soon